nucleus/dataset.py

```diff
@@ -6,11 +10,30 @@ class Dataset:
     """
 
     def __init__(self, dataset_id: str, client):
-        self.dataset_id = dataset_id
+        self.id = dataset_id
         self._client = client
 
     @property
-    def info(self) -> dict:
+    def name(self) -> str:
+        return self._info().get("name", "")
```

Comment:
All of these string literals should use constants instead, I believe many of them already exist in the constants.py file